### PR TITLE
Development version 2024.4.0 (20240717)

### PR DESCRIPTION
**Pull request by mryzhov**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mryzhov** [comment]
evelopment version 2024.4.0 (20240717)

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Review by Wovchena — APPROVED** [review]

**Review by Wovchena — APPROVED** [review]

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -501,6 +501,7 @@ outputs:
         - ccache
         - cmake
         - pybind11
+        - setuptools
```
Comment (mryzhov) [inline recipe/meta.yaml:504]:
```suggestion
```

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,37 +1,40 @@
...
     folder: src/plugins/intel_npu/thirdparty/level-zero  # [linux64]
-  - url: https://github.com/intel/level-zero-npu-extensions/archive/d490a130fbb80e600b3aed3886c305abcb60d77c.tar.gz  # [linux64]
-    sha256: 05c6927fdb338efa4cd17a2f5fe3fb0785a5f5c2ef3be02c51dc9e1e9163cd12  # [linux64]
+  - url: https://github.com/intel/level-zero-npu-extensions/archive/518d64125521cd0f8c98d65f9a0fb40013e95d15.tar.gz  # [linux64]
+    sha256: b161b365decd210ff75abb983887178f2847c56dd733e2128a9f5d899a05c7ba  # [linux64]
     folder: src/plugins/intel_npu/thirdparty/level-zero-ext  # [linux64]
   - url: https://github.com/onnx/onnx/archive/refs/tags/v1.15.0.tar.gz
     sha256: c757132e018dd0dd171499ef74fca88b74c5430a20781ec53da19eb7f937ef68
     folder: thirdparty/onnx/onnx
+  - url: https://github.com/gflags/gflags/archive/refs/tags/v2.2.2.tar.gz
```
Comment (artanokhov) [inline recipe/meta.yaml:35]: gflags is available in Conda-forge

Comment (mryzhov) [reply recipe/meta.yaml:35]: for dev version that is not important, will update the build dependencies later

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 3]
```diff
@@ -387,6 +387,7 @@ outputs:
         - cmake
       host:
         - {{ pin_subpackage('libopenvino', exact=True) }}
+        - gflags
```
Comment (ilya-lavrenov) [inline recipe/meta.yaml:390]: why is it required? only for tools?
I vote for excluding tools from conda build then

Comment (mryzhov) [reply recipe/meta.yaml:390]: no, after excluding the tools from the build

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Review by ilya-lavrenov — APPROVED** [review]

File: recipe/meta.yaml [hunk 4]
```diff
@@ -386,6 +386,7 @@ outputs:
         - {{ stdlib("c") }}
         - cmake
       host:
+        - pugixml
```
Comment (ilya-lavrenov) [inline recipe/meta.yaml:389]: looks like TBB is required as well

**Pull request merged**